tests/Aspire.Hosting.Testing.Tests/TestingBuilderTests.cs

```diff
@@ -13,6 +13,35 @@ namespace Aspire.Hosting.Testing.Tests;
 
 public class TestingBuilderTests
 {
+    [Theory]
+    [RequiresDocker]
+    [InlineData(false)]
+    [InlineData(true)]
+    public async Task CreateAsyncWithOptions(bool genericEntryPoint)
```

Comment:
I'm not sure what a good test might be for this. This is a modification of the existing `HasEndpoints` test.